flare/gp_algebra.py

```diff
@@ -101,7 +101,7 @@ def get_ky_and_hyp_par(hyps: np.ndarray, training_data: list,
     # initialize matrices
     size = len(training_data) * 3
     k_mat = np.zeros([size, size])
-    hyp_mat = np.zeros([size, size, number_of_hyps])
+    hyp_mat = np.zeros([number_of_hyps, size, size])
```

Comment:
nice. It should be noted in the pull request that the hyp_mat slicing has been changed